frontend/src/components/atoms/Routes.tsx

```diff
@@ -17,6 +18,18 @@ import Page from '../templates/Page';
 const Routes: React.FC<RouteComponentProps<{}>> = props => {
   const user = useAuthState();
 
+  /**
```

Comment:
This bug fix should be its own pull request.

Could you open a separate issue for this bug, and then create a new pull request with this fix?

Reply:
Done #169.